### DISCUSSION TITLE
Cant use cx_freeze after adding pyqtgraph

### DISCUSSION THREAD

**Discussion by Ian-Re** [marcelotduarte/cx_Freeze#991] [Q&A] [answered]
Hey everyone,

I'm using cx-freeze==6.5.3 and pyqtgraph==0.11.1.

I can run my application/gui without any problems in pycharm/terminal, however after using cx_freeze with inno setup compiler it wont run and outputs the following (removed personal info):

> 
> C:\Program Files (x86)\TestSuite>HMI.exe
> Traceback (most recent call last):
>   File "C:\Users\User\Desktop\Projects\Software\HMI\env\lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 74, in run
>     module.run()
>   File "C:\Users\User\Desktop\Projects\Software\HMI\env\lib\site-packages\cx_Freeze\initscripts\Console.py", line 36, in run
>     exec(code, m.__dict__)
>   File "main.py", line 4, in <module>
>   File "C:\Users\User\Desktop\Projects\Software\HMI\views\view.py", line 2, in <module>
>     from views.TestSuite import Ui_TestSuite
>   File "C:\Users\User\Desktop\Projects\Software\HMI\views\TestSuite.py", line 11, in <module>
>     from pyqtgraph import PlotWidget
>   File "C:\Users\User\Desktop\Projects\Software\HMI\env\lib\site-packages\pyqtgraph\__init__.py", line 202, in <module>
>     from .graphicsItems.VTickGroup import *
>   File "C:\Users\User\Desktop\Projects\Software\HMI\env\lib\site-packages\pyqtgraph\graphicsItems\VTickGroup.py", line 7, in <module>
>     from .. import functions as fn
>   File "C:\Users\User\Desktop\Projects\Software\HMI\env\lib\site-packages\pyqtgraph\functions.py", line 18, in <module>
>     from . import debug, reload
> ImportError: cannot import name 'debug' from partially initialized module 'pyqtgraph' (most likely due to a circular import) (C:\Program Files (x86)\TestSuite\lib\pyqtgraph\__init__.pyc)

This is strange as I only import pyqtgraph in one file in my code. I dont quite understand what would cause this. Removing pyqtgraph from my application results in a working output from cx_freeze

**Comment by marcelotduarte** [comment, marked as answer]
packages = ["pyqtgraph"] should solve this.
https://cx-freeze.readthedocs.io/en/latest/distutils.html#build-exe

**Reply by marcelotduarte** [reply]
cx_Freeze 6.6 has just been released and supports this as a hook.
